### DISCUSSION TITLE
开源软件项目的定性和定量分析指标 ———— CHAOSS 指标解析 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#119] [Announcements]
# 开源软件项目的定性和定量分析指标 ———— CHAOSS 指标解析 - 「开源之道」

https://opensourceway.community/posts/community_leadership_development/chaoss_metrics_201908_learning/

**Comment by binary5** [comment]
感觉Linux基金会做事一步一个脚印的，挺好。CHAOSS把指标建立和软件建设分成了几个步骤去走：建立指标的时候只需考虑是否能反映开源健康度，不用考虑定量分析是否有难度；定量分析的时候只要考虑数学模型是否能确切表达出指标，不用考虑指标怎么来的；软件建设只需要考虑收集定量数据、实现数学模型，并且提供接口或可视化，给决策者展示或者提供给下游工具访问。目前国内的OpenDigger使用了CHAOSS的部分指标，然后自建数学模型、自建软件，提供了数据查询接口；OSS-Compass使用了CHAOSS的部分指标，借用了CHAOSS体系中的一个软件作为基础，优化了数学模型，提供便于访问的可视化网站。这些都是CHAOSS思想落地实践的案例。
